### PR TITLE
Spruce up local dev env

### PR DESCRIPTION
**Pull request by arecvlohe**
## Background

- The app would crash for a few reasons that were usually pretty simple. One was related to not having a dep installed (slug) and the other was not having a folder available (unprocessed). (4da839e, 4481728, b5ed15c)
- There were some formatting inconsistencies that were also cleaned up with the help of prettier (" vs ' and ; vs no ;)(58046b5). 
- I ran the app to see if anything had changed and it seemed there were changes. Some files were updated but only in ways that are not noticable (51153eb).
Screen Shot 2020-11-27 at 6 57 02 PM

- Ran the images through [imageOptim](https://imageoptim.com) to reduce their file size in most cases by 50% (706776e)
Screen Shot 2020-11-27 at 6 52 53 PM

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,4 @@
 node_modules/
-unprocessed/*
\ No newline at end of file
+# ignore files in unprocessed but keep the unprocessed folder for local dev
+unprocessed/*
+!unprocessed/.keep
```
Comment (nicksahler) [inline .gitignore:4]: Good catch

File: index.js [hunk 2]
```diff
@@ -1,124 +1,130 @@
...
+
+const main = async () => {
+  await scrape_wiki(US_URL, 'us')
+  await scrape_wiki(CA_URL, 'ca')
+
+  await download_images()
+  await resize_images()
+}
+
+main()
```
Comment (nicksahler) [inline index.js:130]: Are there any major programmatic changes here or just prettier? Hard to track with the prettification.

**Review by nicksahler — APPROVED** [review]
LGTM!

File: index.js (hunk 2 above)
Comment (arecvlohe) [reply index.js:130]: No programmatic changes, only prettier

File: .gitignore (hunk 1 above)
Comment (arecvlohe) [reply .gitignore:4]: Just need to add the folder and put .keep in it ;). Made the change here: b5ed15c

File: index.js (hunk 2 above)
Comment (arecvlohe) [reply index.js:130]: It's easier if you add `?w=1` at the end of the commit url

example: https://github.com/nativesintech/indigemoji/pull/7/commits/58046b5db7bf5dd1413833df4bf6e9f6ea2121f7?w=1

It ignores white-space changes and such and is way easier to read, re: https://github.blog/2011-10-21-github-secrets/#whitespace

Comment (nicksahler) [reply index.js:130]: AH! I see. That helps. I always forget about that.

**Review by nicksahler — APPROVED** [review]

**Pull request merged**